src/PostgREST/Auth/JwtCache.hs

```diff
@@ -58,12 +70,16 @@ lookupJwtCache JwtCacheState{jwtCache} token maxLifetime parseJwt utc = do
 
       let timeSpec = getTimeSpec res maxLifetime utc
 
-      -- purge expired cache entries
-      C.purgeExpired jwtCache
-
       -- insert new cache entry
       C.insert' jwtCache (Just timeSpec) token res
 
+      -- Execute IO action to purge the cache
+      -- It is assumed this action returns immidiately
+      -- so that request processing is not blocked.
+      -- If cache purging is slow it should trigger
+      -- asynchronous purge operation
```

Comment:
Remove unnecessary comment.
```suggestion
      -- Execute IO action to purge the cache
      -- It is assumed this action returns immediately
      -- so that request processing is not blocked.
```